terraform/modules/ecr/outputs.tf

```diff
@@ -1,3 +1,15 @@
 output "image_uri" {
   value = data.aws_ecr_image.this.image_uri
 }
+
+output "repo_name" {
+  value = module.ecr.repository_name
+}
+
+output "repo_url" {
+  value = module.ecr.repository_url
+}
+
+output "repo_arn" {
```

Comment:
This will get used when the ingestion lambda points to the new ECR and needs to depend on that resource